portality/forms/application_forms.py

```diff
@@ -2879,3 +2879,39 @@ def wtform(formulaic_context, field, wtfargs):
 
 ApplicationFormFactory = Formulaic(APPLICATION_FORMS, WTFORMS_BUILDERS, function_map=PYTHON_FUNCTIONS, javascript_functions=JAVASCRIPT_FUNCTIONS)
 JournalFormFactory = Formulaic(JOURNAL_FORMS, WTFORMS_BUILDERS, function_map=PYTHON_FUNCTIONS, javascript_functions=JAVASCRIPT_FUNCTIONS)
+
+
+if __name__ == "__main__":
```

Comment:
Can you add a few comments to explain the documention steps here and expected use?

Reply:
I've opened a new issue to do that, since this one is merged: https://github.com/DOAJ/doajPM/issues/2918